build-ps/jenkins/build-binary

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o xtrace
+
+
+# ------------------------------------------------------------------------------
+# Declate all input variables
+# ------------------------------------------------------------------------------
+JOB_CMAKE=${JOB_CMAKE:-cmake}
+COMPILER=${COMPILER:-default}
+CMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE:-RelWithDebInfo}
+WITH_TOKUDB=${WITH_TOKUDB:-ON}
+WITH_ROCKSDB=${WITH_ROCKSDB:-ON}
+WITH_EMBEDDED_SERVER=${WITH_EMBEDDED_SERVER:-ON}
+WITH_RAPID=${WITH_RAPID:-ON}
+WITH_SSL=${WITH_SSL:-system}
+WITHOUT_PERFSCHEMA_STORAGE_ENGINE=${WITHOUT_PERFSCHEMA_STORAGE_ENGINE:-OFF}
+ANALYZER_OPT=${ANALYZER_OPT:-}
+MAKE_OPTS=${MAKE_OPTS:--j$(grep -c ^processor /proc/cpuinfo) VERBOSE=1}
+CMAKE_OPTS=${CMAKE_OPTS:-}
+BUILD_COMMENT=${BUILD_COMMENT:-}
+TAG=${TAG:-}
+DIST_NAME=${DIST_NAME:-}
+SSL_VER=${SSL_VER:-}
+
+
+# ------------------------------------------------------------------------------
+# set working dir
+# ------------------------------------------------------------------------------
+SOURCEDIR=$(cd $(dirname $0)/../..; pwd -P)
+WORKDIR=$(cd ${1:-.}; pwd -P)
+INSTALL_DIR=${SOURCEDIR}/DESTDIR
+mkdir -p ${WORKDIR} ${INSTALL_DIR} ${WORKDIR}/libboost 
+
+
+# ------------------------------------------------------------------------------
+# Set OS/Arch flags
```

Comment:
The whole purpose of this is to build for i686 rather than i386 (default) for 32-bit platforms.
This should be done by adding
`-DCMAKE_C_FLAGS="-m32 -march=i686" -DCMAKE_CXX_FLAGS="-m32 -march=i686"` to CMake config options for 32-bit platforms,

Reply:
we have such transformation in cmake command